applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -114,8 +114,17 @@ class MapperVertexMorphing
         CreateListOfNodesOfDesignSurface();
         CreateFilterFunction();
         InitializeMappingVariables();
-        AssignMappingIds();        
-        ComputeMappingMatrix();
+        AssignMappingIds();
+
+        // optional flags
+        try {
+            mConsistentBackwardMapping = optimizationSettings["design_variables"]["consistent_backward_mapping"].GetBool();
+        }
+        catch (...)
+        {
+            mConsistentBackwardMapping = false;
+        }
```

Comment:
Use of try catch for handling "normal" behavior of the program is not a good idea at all, and should be avoided. The try and catch should be used only for exceptional cases.